examples/http-server/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/corazawaf/coraza/v3/examples/http-server
 
-go 1.20
+go 1.22
```

Comment:
while this is great it break our policy of 2 latest versions supported.

Reply:
Sorry, wasn't aware of the policy.  I can make this work without this change.

Reply:
resolved via PR #1145